src/perf/lib/PerfServer.h

```diff
@@ -15,6 +15,58 @@
 #include "SecNetPerf.h"
 #include "Tcp.h"
 
+class PerfServer;
+struct StreamContext;
+
+typedef enum SYNTHETIC_DELAY_TYPE {
+    SYNTHETIC_DELAY_FIXED,
+    SYNTHETIC_DELAY_VARIABLE
+} SYNTHETIC_DELAY_TYPE;
+
+
+struct DelayedWorkContext {
+    StreamContext* Context;
+    void* Handle;
+    bool IsTcp;
+    DelayedWorkContext* Next;
+};
```

Comment:
You can delete this entire struct now if you simply put a `DelayNext` pointer in the `StreamContext` itself.

Reply:
I strongly prefer we remove this. If we ever need more metadata, we can simply but it on the stream like everything else. In general, I want to apply the principle of keeping code simple now unless there is strong need/plan to add something soon in the future that cannot be supported with the simple design.